h2o-py/tests/testdir_misc/pyunit_ice_plot.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function
+
+import sys
+
+sys.path.insert(1,"../../")
+import matplotlib
+matplotlib.use("Agg")  # remove warning from python2 (missing TKinter)
+import h2o
+import matplotlib.pyplot
+from tests import pyunit_utils
+from h2o.estimators import *
+
+
+def test_display_mode():
+    train = h2o.upload_file(pyunit_utils.locate("smalldata/titanic/titanic_expanded.csv"))
+    y = "survived"
+
+    # get at most one column from each type
+    cols_to_test = []
+    for col, typ in train.types.items():
+        for ctt in cols_to_test:
+            if typ == train.types[ctt] or col == y:
+                break
+        else:
+            cols_to_test.append(col)
+
+    gbm = H2OGradientBoostingEstimator(seed=1234, model_id="my_awesome_model")
+    gbm.train(y=y, training_frame=train)
+
+    assert isinstance(gbm.ice_plot(train, 'title', binary_score_form="logodds").figure(), matplotlib.pyplot.Figure)
+    assert isinstance(gbm.ice_plot(train, 'age').figure(), matplotlib.pyplot.Figure)
+    matplotlib.pyplot.close("all")
```

Comment:
I would also test the incorrect inputs (regression model, unknown string in `binary_score_form`), to be sure everything works correctly.

Reply:
[b860661](https://github.com/h2oai/h2o-3/pull/6034/commits/b8606617b883959d375cf405435bd6898b2211b4)